internal/dag/builder.go

```diff
@@ -518,6 +521,18 @@ func (b *Builder) computeHTTPProxy(proxy *projcontour.HTTPProxy) {
 			svhost.Secret = sec
 			svhost.MinProtoVersion = annotation.MinProtoVersion(proxy.Spec.VirtualHost.TLS.MinimumProtocolVersion)
 
+			if proxy.Spec.VirtualHost.TLS.EnableFallbackCertificate && !isBlank(b.FallbackCertificate) {
+				secretName := splitSecret(b.FallbackCertificate, "projectcontour")
+				sec, err = b.lookupSecret(secretName, validSecret)
+				if err != nil {
+					sw.SetInvalid("FallbackCertificate requested, but Certificate Secret %q is invalid or missing", b.FallbackCertificate)
```

Comment:
This status message should follow the conventions of the other messages, so something like:
```Go
sw.SetInvalid("Spec.Virtualhost.TLS fallback certificate Secret %q is invalid: %s", b.FallbackCertificate, err)
```